tokio-console/src/view/mod.rs

```diff
@@ -96,6 +96,15 @@ impl View {
     pub(crate) fn update_input(&mut self, event: input::Event, state: &State) -> UpdateKind {
         use ViewState::*;
         let mut update_kind = UpdateKind::Other;
+
+        if matches!(event, key!(Char('t'))) {
+            self.state = TasksList;
+            return update_kind;
+        }
+        if matches!(event, key!(Char('r'))) {
+            self.state = ResourcesList;
+            return update_kind;
+        }
```

Comment:
very minor style nit: i would add a line of whitespace here:
```suggestion
        }
        
        if matches!(event, key!(Char('r'))) {
            self.state = ResourcesList;
            return update_kind;
        }
        
```